lib/theme-options/live-options.php

```diff
@@ -0,0 +1,30 @@
+<?php
+// SITE OPTIONS
+$prefix = '_igv_';
+$suffix = '_options';
+
+$page_key = $prefix . 'live' . $suffix;
+$page_title = 'Live Page Options';
+$metabox = array(
```

Comment:
I can't really tell. Other thing is that maybe all the home/live/sound&vision pages options should be in the same place. Then basically all the editorial configuration of the site happens in Theme Options which I like as a UX pattern. Rather than finding each page in pages which doesn't quite feel right even tho thats how the WP data works